### PR TITLE
Check SQL table and column names

### PR DESCRIPTION
**Pull request by rco-odoo**

### CODE REVIEW

**Comment by rco-odoo** [comment]
@xmo-odoo what do you think about this?

File: openerp/models.py [hunk 1]
```diff
@@ -60,6 +60,7 @@
 
 regex_order = re.compile('^(\s*([a-z0-9:_]+|"[a-z0-9:_]+")(\s+(desc|asc))?\s*(,|$))+(?<!,)$', re.I)
 regex_object_name = re.compile(r'^[a-z0-9_.]+$')
+regex_pg_name = re.compile(r'[a-z_][a-z0-9_$]*', re.I)
```
Comment (guewen) [inline openerp/models.py]: Shouldn't it be `re.compile(r'^[a-z_][a-z0-9_$]*$', re.I)`?
It seems to me that `regex_pg_name.match('abc!') returns a group with 'abc' with the current regex.

Comment (rco-odoo) [inline openerp/models.py]: @guewen You are right!
Gonna fix it.

Comment (rco-odoo) [inline openerp/models.py]: See #8412

Comment (rco-odoo) [inline openerp/models.py]: @guewen I improved the regexp just like you suggested me.
Thank you very much!

Comment (guewen) [inline openerp/models.py]: You're welcome :blush:

**Pull request merged**